app/assets/stylesheets/application.css.scss

```diff
@@ -202,3 +202,16 @@ main {
   color: white;
   box-shadow: 1px 1px 10px 1px rgba(0,0,0,0.7);
 }
+
+.town_table td{
+  border: none;
+  border-collapse: collapse;
+  border-spacing: 0px;
+  padding-bottom: 0px
+}
+
+.town_table img.responsive-img {
+    max-width: inherit !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered display, max-width, vertical-align